packages/api/src/post-build.js

```diff
@@ -105,20 +135,50 @@ class PostBuildHandler {
     }
   }
 
+  static async flattenTap (datastring) {
+    const output = await new Promise(resolve => {
+      var process = cp.exec('tap-parser -f -t', (error, stdout, stderr) => {
+        if (error || stderr) {
+          throw new InvalidParameterError('Could not parse and flatten tap');
+        }
+        resolve(stdout);
+      });
+
+      // write your multiline variable to the child process
+      process.stdin.write(datastring);
+      process.stdin.end();
+    });
+    return output.replace(/\d+..\d+\n/, '\n'); // hacky solution to issue where 1..0 shows up at beginning
+  }
+
+  static removeDuplicateTestCases (testCases) {
+    const testCasesNew = [];
```

Comment:
I usually name this something like `result`. That way, it's clear you're building this list up and plan to return it.